config/routes.rb

```diff
@@ -2,6 +2,8 @@
   resources :locators, only: :index
   resources :locations, only: :index
 
+  post 'twilio' => 'twilios#show'
```

Comment:
As discussed this is to keep the code inline with the current implementation. We should look to move to get requests once it is live.